backend/tests/authentication.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment node
+ */
+const axios = require('axios');
+const mongoose = require('mongoose');
+const usersModel = require('../models/users');
+
+// Setup
+const baseURL = 'http://localhost:3001';
+beforeAll(async () => {
+  const dbuser = process.env.dbuser;
+  const dbpass = process.env.dbpass;
+  mongoose.set('useFindAndModify', false);
+  await mongoose.connect('mongodb+srv://' + dbuser + ':' + dbpass + '@cloudhaven.92yac.mongodb.net/CloudHaven?retryWrites=true&w=majority', {useNewUrlParser: true});
+});
+
+// Verify user exists in the database
+test('should verify if the user already exists in DB', async () => {
+  return await axios.get(baseURL)
+      .then((response) => {
+        expect(response.data)
+            .toEqual({username: 'sample_user1' password: password});
```

Comment:
```suggestion
            .toEqual({username: 'sample_user1', password: password});
```
You missed a comma here